CMakeLists.txt

```diff
@@ -2,45 +2,55 @@ cmake_minimum_required(VERSION 3.1)
 project(bspline)
```

Comment:
can you change project name to splx lol. this is a legacy name from 7 months ago :dagger: